pkgs/by-name/tf/tftui/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "tftui";
+  version = "0.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "idoavrah";
+    repo = "terraform-tui";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7nNR9mueSEoO1sNxazyL4ViiO7e0vlGSINCCyaU/f1s=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "version = \"x.x.x\"" "version = \"${version}\""
+  '';
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
```

Comment:
Not sure if this should have `terraform` as propagated build input? When testing this, it failed as I didn't had terraform in path.

Reply:
I suspect having a default would make usage more cumbersome for everyone who doesn't want it. I did consider this but decided against it as I usually use `terraform.withPlugins` and Terragrunt on a per-project basis so I never actually want the default Terraform from Nixpkgs. It seemed that this most flexible approach was for the user to ensure that something called `terraform` with Terraform-like behaviour (ie. Terragrunt) is available in their `PATH`.

Reply:
Yes, makes sense.